<!--
@author:    Johann Schmidt
@date:      November 2019
@brief:     Script used to spawn a turtlebots.
@todo:
-->

<launch>
    <node pkg="benchmark" type="spawning_controller.py" name="spawning_controller" output="screen" clear_params="true">
        <param name="model_type" type="str" value="burger" />
        <param name="number_of_robots" type="int" value="4" />
        <param name="namespace" type="str" value="tb3_" />
    </node>
</launch>